src/lib/core/plugin.js

```diff
@@ -160,8 +160,12 @@ Plugin.prototype.addContractFile = function(file) {
   this.addPluginType('contractFiles');
 };
 
-Plugin.prototype.registerConsoleCommand = function(cb) {
-  this.console.push(cb);
+Plugin.prototype.registerConsoleCommand = function(optionsOrCb) {
+  if (typeof optionsOrCb === 'function') {
+    this.logger.warn(__('Registering console commands with a function is deprecated'));
+    // TODO add docs on how to register
```

Comment:
I don't think we want to keep this comment here.

Reply:
That's a reminder to me/us that when we write docs on the new way of registering, to have a link to the docs

Reply:
Understood, but I don't think that comment helps in the source code. Then I'd say rather open an issue on embark-site and put a comment in there that the warning should be updated in the source as well.